cmd/test.go

```diff
@@ -252,6 +253,17 @@ func opaTest(args []string) int {
 			}
 		}
 	} else {
+		//for test coverage with --bundle, retrieve modules from bundles
+		if modules == nil {
+			modules := map[string]*ast.Module{}
```

Comment:
This is not doing what you expect -- it's declaring a new variable called `modules` that will not be visible outside of this block.

Reply:
re declared and initialised `modules` variable outside the block